src/App.js

```diff
@@ -62,7 +66,15 @@ function App() {
               </Suspense>
             </RequiredAuth>}
           />
+          <Route path={ROUTES.ROUTE_PATH_POSTPAGE} element={
+            <RequiredAuth>
+              <Suspense fallback={"..loading"}>
+                <PostPage />
```

Comment:
write fallback component it will be good ui and experience

Reply:
will do that once bare minimum is done with all p0